development/cluster/create-env.sh

```diff
@@ -1,18 +1,45 @@
 #!/usr/bin/env bash
-set -e
+# Usage:
+# To disable metrics: development/minikube.sh
+# To enable metrics: METRICS=1 development/minikube.sh
+set -o errexit  # Used to exit upon error, avoiding cascading errors
+set -o pipefail # Unveils hidden failures
+set -o nounset  # Exposes unset variables
 
-echo "========> creating cluster..."
-minikube start --profile kgw --addons=metallb
+PROFILE="kgw"
+
+function print_separator() {
+  local header="${1} "
+  local separator_chars=""
+  separator_chars=$(printf -- '-%.0s' $(seq 1 $(($(tput cols) - ${#header}))))
+  printf "%b" "$(
+    tput bold
+    tput setaf 2
+  )" "${header}" "${separator_chars}" "$(tput sgr0)" "\n"
+}
+
+print_separator "creating cluster"
```

Comment:
What's the output from this? Just curious what print_separator actually does

Reply:
```sh
Deploying default Envoy Fleet ------------------------------------------------------------------------------------------
make[1]: Entering directory '/media/home/work/kubeshop.io/kusk-gateway'
kubectl apply -f config/samples/gateway_v1_envoyfleet.yaml
envoyfleet.gateway.kusk.io/default created
make[1]: Leaving directory '/media/home/work/kubeshop.io/kusk-gateway'
make create-env  14.00s user 4.28s system 17% cpu 1:43.93 total
```

![image](https://user-images.githubusercontent.com/22658/181261319-c8190024-5984-4e3a-ae9b-89ce572a28e4.png)

It prints green text and `-` separators until it reaches the width of the terminal.